profile/CSIP.xml

```diff
@@ -1387,7 +1387,7 @@
             xsi:schemaLocation="http://www.loc.gov/METS/ http://www.loc.gov/standards/mets/mets.xsd http://www.w3.org/1999/xlink http://www.loc.gov/standards/mets/xlink.xsd https://dilcis.eu/XML/METS/CSIPExtensionMETS https://dilcis.eu/XML/METS/CSIPExtensionMETS/DILCISExtensionMETS.xsd">
          </mets:mets>
     </Example>
-    <Example ID="metsRootElementExample3" LABEL="METS root element example with transferred content of a type from the fixed value list and the representation content is not part of the value list, but is instead a value decided upon in the submission agreement.">
+    <Example ID="metsRootElementExample2" LABEL="METS root element illustrating the use of a custom csip:@OTHERCONTENTINFORMATIONTYPE attribute value when an appropriate content type value could not be found in the vocabulary. The csip:@CONTENTINFORMATIONTYPE attribute value is set to OTHER.">
```

Comment:
```suggestion
    <Example ID="metsRootElementExample2" LABEL="METS root element illustrating the use of a custom csip:@OTHERCONTENTINFORMATIONTYPE attribute value when the correct value for the Content Information Type Specification does not exist in the vocabulary (the csip:@CONTENTINFORMATIONTYPE attribute value is set to OTHER).">
```

I made the wording more strict, as CITS is a very specific document that has to be referenced precisely. "Appropriate value could not be found" sounds as if it was OK to find an approximate match.